lib/Activity.py

```diff
@@ -1108,6 +1108,19 @@ def watch_next_wf(self, sessionid):
                 if tar_path and os.path.exists(tar_path):
                     # The path exists and process_media has been run.  Note
                     # the directory will not exist before process-media is ran.
+                    if os.path.isfile(tar_path):
+                        # All files in the tar file was not extracted into a new directory.
+                        # Files were extracted to the media dir.
+                        # Assuming this is a Docs tar file.
+                        file_tar_msg = formatted(f"""
+                            Tar file {product} found in media directory
+                            was not extracted into a new directory.
+                            Some files were extracted into the media
+                            directory. Assuming this is a Docs tarfile
```

Comment:
`s/is a Docs tarfile/is not an IUF tarfile/`

Reply:
changed to 'Assuming this is not an IUF product tarfile'